pkg/cmd/attestation/verify/verify_test.go

```diff
@@ -463,4 +463,41 @@ func TestRunVerify(t *testing.T) {
 		customOpts.BundlePath = ""
 		require.Error(t, runVerify(&customOpts))
 	})
+
+	t.Run("with valid OCI artifact", func(t *testing.T) {
```

Comment:
Can I just double check that none of these tests are going to make network connections?